core/templates/dev/head/domain/exploration/RecordedVoiceoversObjectFactory.ts

```diff
@@ -16,22 +16,34 @@
  * @fileoverview Factory for creating new frontend instances of
  * RecordedVoiceovers domain objects.
  */
-
+export interface RecordedVoiceOverBackendInterface {
+  'voiceovers_mapping': {
+    [propName: string]: {
+      [propName: string]: VoiceoverInterface
+    }
+  }
+}
+export interface voice {
```

Comment:
I think interface name should start with a capital letters.

Reply:
changed it in the recent commit